cmd/domain.go

```diff
@@ -71,6 +88,19 @@ func domainScannerLogic(ctx *cobra.Command, readr io.ReadWriteSeeker) []rdap.Who
 	return result
 }
 
+func getRootDomain(domain string) (string, error) {
+	// Extract root domain from provided domain.
+	rootDomain, err := publicsuffix.EffectiveTLDPlusOne(domain)
+	if err != nil {
+		return rootDomain, err
+	}
+	// Check to see if root domain is different than domain provided.
+	if rootDomain != domain {
+		cli.Debug("Using Root Domain '%v' instead of '%v'", rootDomain, domain)
+	}
+	return rootDomain, err
```

Comment:
The valid return should return rootDomain, nil

Reply:
Updated with additional commit to resolve issue